cmd/grpcServer/main.go

```diff
@@ -43,17 +43,25 @@ func main() {
 	// @todo: this grpcServer.GracefulStop()
 
 	// decide which function to run
-	var greeter grpcserver.GreetProvider = _greeter.New()
-	if strings.EqualFold(os.Getenv(envGreeter), "db") { // picked up by the linter, this is func ignores case
+	var greeter grpcserver.GreetProvider
+	switch os.Getenv(envGreeter) {
```

Comment:
wrap the `os.Getenv` call in `strings.ToLower` saves user confusion as they never read the documentation properly

Reply:
Done, cheers